### PR TITLE
Update docs to reflect inclusion of  data-action attribute

### PR DESCRIPTION
**Pull request by eballantine**
This is so the frontend team can more easily target these forms. Suggestion to update docs further with examples of how we intend to use this targeting

### CODE REVIEW

**Comment by eballantine** [comment]
2 suggestions;
1) Either examples or just a note below the 'output' pointing to the data attribute to suggest it's use. 
2) Should 'form tag' be it's own step in the navigation? Or a subtitle in the tags index just to show hey there's this group of tags that behaves slightly differently?

File: docs/reference/tags/form_tags/clear_cart/index.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ The `clear_cart` tag renders a button that clears the customer's cart.
 ##### output
 {% raw %}
 ```html
-<form action="/sites/cart" accept-charset="UTF-8" method="post">
+<form action="/sites/cart" date-action="clear_cart" accept-charset="UTF-8" method="post">
```
Comment (samantha-lind) [inline docs/reference/tags/form_tags/clear_cart/index.md:24]: @eballantine should this be `data-action` rather than `date-action`?

Comment (eballantine) [reply docs/reference/tags/form_tags/clear_cart/index.md:24]: Thanks @samantha-lind will fix

Comment (geoffreymm) [reply docs/reference/tags/form_tags/clear_cart/index.md:24]: @eballantine we are going to use `data-form-action` instead, just to ensure it doesn't clash with other js that we currently use that format.

**Comment by geoffreymm** [comment]
[Update return of liquid form tags to include helpful data attributes](https://trello.com/c/UBnfg7lH/55-update-return-of-liquid-form-tags-to-include-helpful-data-attributes)

**Review by geoffreymm — APPROVED** [review]

**Pull request merged**